src/pybind/mgr/dashboard/frontend/src/app/shared/components/form-modal/form-modal.component.html

```diff
@@ -83,10 +83,12 @@
                          [formControlName]="field.name"
                          itemValueKey="content"
                          [id]="field.name"
-                         [items]="field?.typeConfig?.options"
                          [invalid]="getError(field)"
                          [invalidText]="getError(field)"
                          [appendInline]="false"
+                         cdDynamicInputCombobox
+                         (updatedItems)="field.typeConfig.options = $event"
```

Comment:
are we just assigning here ?, want to confirm just.

Reply:
yup